scholly/base/src/layouts/layout1/App.js

```diff
@@ -8,6 +8,15 @@ import { TEAM_INFO } from '../../global/data';
 import Header from './components/layout/Header';
 import getRoutes from './components/router/routes';
 import './components/centered.css';
+import '../../shared/css/style.css';
+import '../../shared/css/baseColours.css';
+
+const darkMode = false;
```

Comment:
Can you move the `darkMode = false` or something like `themeOption` into `fakeData.js` and import it here like how we import the layoutOption? I think that would be easier for others who are currently working on scholly frontend.

Reply:
@jfan0010 Kunj has added a FAKE_TEAM_SITE_METADATA in PR 226.
Didn't want to conflict with that, so I think it's better for me to use that after PR 226 is merged.

Reply:
@jfan0010 I've changed the boolean into a string for now anyway, so it will be more consistent with what Kunj is implementing. Although still haven't used fakeData.js for now to not conflict with PR 226 changes.